extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -819,6 +819,11 @@ export class ProjectsController {
 			return; // user cancelled
 		}
 
+		// Check if itemObjectName contains the file extension, remove it if so
+		while (itemObjectName.toLowerCase().endsWith(fileExtension.toLowerCase())) {
```

Comment:
Odd for this to be a `while`; I think it should be just `if`.  I understand the thought, but IMO if the user intentionally types `MyTable.sql.sql`, I think the more predictable behavior is to only use the "last" matching extension.